src/RBACHandler.test.ts

```diff
@@ -206,114 +236,228 @@ describe('isAuthorized:Export', () => {
         },
     });
 
+    // const getBaseResource = (fhirVersion: string) => {
+    //     return fhirVersion === '3.0.1' ? BASE_STU3_RESOURCES : BASE_R4_RESOURCES;
+    // };
+    //
+    // each(['3.0.1', '4.0.1']).test('TRUE:%s: GET system Export with permission to all resources', fhirVersion => {

```

Comment:
I tried to see what the tests would look like if we used `each`. The difference is we would use a `fhirVersion` array for each test. I felt like this format is more verbose than the `forEach` format currently written in the test. What are your thoughts on the `each` format versus the `forEach` format?

Reply:
`jest-each` has many features. With the right setup it shouldn't result in more verbose code

I think that here you'd want to use `each(['3.0.1', '4.0.1']).describe()` here

https://github.com/facebook/jest/tree/master/packages/jest-each

Still, It's not mandatory to use `jest-each`